Description: Make -maltivec optional on Powerpc to fix powerpcspe FTBFS
Author: Roland Stigge <stigge@antcom.de>
Bug-Debian: http://bugs.debian.org/701590
Forwarded: https://github.com/lightspark/lightspark/pull/216
Origin: vendor
Last-Update: 2013-02-28

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -142,7 +142,10 @@
 ELSEIF(${CMAKE_SYSTEM_PROCESSOR} MATCHES "ppc")
 	SET(ppc 1)
 	SET(LIB_SUFFIX "" CACHE STRING "Choose the suffix of the lib folder (if any) : None ppc")
-	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -maltivec -D__STDC_CONSTANT_MACROS -fPIC")
+	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D__STDC_CONSTANT_MACROS -fPIC")
+	IF(NOT(DISABLE_ALTIVEC))
+		SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -maltivec")
+	ENDIF()
 ELSE(${CMAKE_SYSTEM_PROCESSOR} MATCHES "^i[3-6]86$|^x86$")
 	# All non-x86 non-ppc architectures.
 	SET(LIB_SUFFIX "" CACHE STRING "Choose the suffix of the lib folder (if any) : None")
